File: backup/backup-2021-09-15/bl-merchandise-data-collector/src/main/java/org/jesperancinha/logistics/mcs/MerchandiseControlService.java
package org.jesperancinha.logistics.mcs;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@EnableJpaRepositories("org.jesperancinha.logistics.jpa.repositories")
@EntityScan("org.jesperancinha.logistics.jpa.model")
public class MerchandiseControlService {
    public static void main(String[] args) {
        SpringApplication.run(MerchandiseControlService.class, args);
    }
}

File: bl-central-server/bl-merchandise-data-collector/src/main/kotlin/org/jesperancinha/logistics/mcs/MerchandiseControlService.kt
package org.jesperancinha.logistics.mcs

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication
@EnableJpaRepositories("org.jesperancinha.logistics.jpa.dao")
@EntityScan("org.jesperancinha.logistics.jpa.dao")
class MerchandiseControlService

fun main(args: Array<String>) {
    SpringApplication.run(MerchandiseControlService::class.java, *args)
}